src/builtins/BuiltinTemporal.cpp

```diff
@@ -197,6 +204,139 @@ static Value builtinTemporalPlainDateInLeapYear(ExecutionState& state, Value thi
     return TemporalCalendar::calendarInLeapYear(state, thisValue.asObject()->asTemporalPlainDateObject()->calendar().asObject(), thisValue);
 }
 
+static Value builtinTemporalPlainTimeConstructor(ExecutionState& state, Value thisValue, size_t argc, Value* argv, Optional<Object*> newTarget)
+{
+    if (!newTarget.hasValue()) {
+        ErrorObject::throwBuiltinError(state, ErrorObject::TypeError, ErrorObject::Messages::New_Target_Is_Undefined);
+    }
+
+    if (!(argc > 0 && argv[0].isInteger(state)) || !(argc > 1 && argv[1].isInteger(state)) || !(argc > 2 && argv[2].isInteger(state)) || !(argc > 3 && argv[3].isInteger(state)) || !(argc > 4 && argv[4].isInteger(state)) || !(argc > 5 && argv[5].isInteger(state))) {
```

Comment:
I don't really like this huge check, maybe a `for` loop could be used here:
```
int values[6];
memset(values, 0, sizeof(values));

for (int i = 0; i < argc && i < 6; i++) {
  if (!(argv[0].isInteger(state))) {
    ErrorObject::throwBuiltinError(state, ErrorObject::TypeError, "Invalid type");
  }
  values[i] = argv[i].asInt32();
}
```

Then the values could be passed to createTemporalTime.